packages/server/utils/OpenAIServerManager.ts

```diff
@@ -41,6 +42,64 @@ class OpenAIServerManager {
     }
   }
 
+  async getDiscussionPromptQuestion(topic: string, reflections: Reflection[]) {
+    if (!this.openAIApi) return null
+    const prompt = `As the meeting facilitator, your task is to steer the discussion in a productive direction. I will provide you with a topic and comments made by the participants around that topic. Your job is to generate a thought-provoking question based on these inputs. Here's how to do it step by step:
+
+    Step 1: Categorize the discussion into one of the following four groups:
+
+    Group 1: Requirement/Seeking help/Requesting permission
+    Example Question: "What specific assistance do you need to move forward?"
+
+    Group 2: Retrospection/Post-mortem/Looking back/Incident analysis/Root cause analysis
+    Example Question: "What were the underlying factors contributing to the situation?"
+
+    Group 3: Improvement/Measurement/Experiment
+    Example Question: "What factors are you aiming to optimize or minimize?"
+
+    Group 4: New plan/New feature/New launch/Exploring new approaches
+    Example Question: "How can we expedite the learning process or streamline our approach?"
+
+    Step 2: Once you have categorized the topic, formulate a question that aligns with the example question provided for that group. If the topic does not belong to any of the groups, come up with a good question yourself for a productive discussion.
+
+    Step 3: Finally, provide me with the question you have formulated without disclosing any information about the group it belongs to.
+
+    Topic: ${topic}
+    Comments:
+    ${reflections
+      .map(({plaintextContent}) => plaintextContent.trim().replace(/\n/g, '\t'))
+      .join('\n')}`
+    try {
+      const response = await this.openAIApi.createChatCompletion({
+        model: 'gpt-3.5-turbo',
+        messages: [
+          {
+            role: 'user',
+            content: prompt
```

Comment:
I copied and pasted the reflections from our last Squad retro and gave it a spin, and I think the prompts are generally very good! 

I did get one that repeated the instructions though:
![Screenshot 2023-07-10 at 20 00 05](https://github.com/ParabolInc/parabol/assets/39854876/21bdc429-f15e-4983-ad14-973110b76278)

And two of them included quotation marks around the discussion topic:

![Screenshot 2023-07-10 at 20 00 32](https://github.com/ParabolInc/parabol/assets/39854876/4fb074f7-c542-446c-ae3f-309ba8c4c7b1)

I think that's fine for internal testing though. The prompt can always be updated.

Reply:
Made some improvements